src/lua/tests.rs

```diff
@@ -0,0 +1,31 @@
+//! Lua thread test.
+//! Spawns a lua thread!
+
+use std::time::Duration;
+use std::thread;
+
+use super::*;
+
+#[test]
+fn big_lua_test() {
+    println!("Testing lua thread...");
+    super::init();
+    thread::sleep(Duration::from_secs(5));
```

Comment:
Well what happens is when you send you get a "Lua thread not initialized". The `sender` channel isn't initialized until the thread is up and running, and the send will also fail if `lua::is_running()` returns false - buffering isn't designed to work while the thread is dead right now (especially because the lua state will be different after a restart).

I think the only use for it would be testing, but we could write a method that busy-waits/sleeps until `lua::is_running()` and then sends.